app/views/site/index.html.erb

```diff
@@ -1 +1,30 @@
 <% provide(:title, "Home") %>
+<div class="jumbotron">
+  <h1>Adopts a gem</h1>
+  <p class="lead">RubyGems needs a way to mark a gem as "maintainer needed" or "someone please take this gem over". This project would drive the creation of an "Adoption Center" where people could list gems that want a maintainer/someone new to help fix issues, cut releases, etc, and then assist in the handoff of the gem.</p>
+  <p><input type="text" class="form-control" placeholder="Search a gem" aria-describedby="basic-addon1"></p>
+</div>
+
+<div class="row marketing">
```

Comment:
I don't think we need this section since we won't be listing gems on the landing page, as we discussed the idea is for this landing page to be similar to [RubyGems.org](https://rubygems.org/) one being simple and having a search field and some other text and relevant links.